nikola/nikola.py

```diff
@@ -564,7 +566,9 @@ def __init__(self, **config):
                                              'social_buttons_code',
                                              'search_form',
                                              'body_end',
-                                             'extra_head_data',)
+                                             'extra_head_data',
+                                             'date_format',
+                                             'js_date_format',)
```

Comment:
I now did the `json.dumps()` before creating the `TranslatableSetting` object creations. This way, I only have to change a bit in `nikola.py`, but not in all places where `_GLOBAL_CONTEXT` is converted to a local context (that happens several times in `nikola.py` and in at least four plugins).